File: Programmers/Level 1/완주하지 못한 선수.py
def solution(participant, completion):
    answer = ''
    
    for i in range(len(participant)):
        if participant[i] not in completion:
            return participant[i]
        else :
            participant.sort()
            completion.sort()
            for j in range(len(completion)):
                if participant[j] != completion[j]:
                    return participant[j]완주

File: Programmers/Level 1/완주하지 못한 선수.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    int i = 0;
        
    sort(participant.begin(), participant.end());
    sort(completion.begin(), completion.end());
    
    for(int k = completion.size() ; k < participant.size() ; k++)
        completion.push_back("NULL");
    
    for(i ; i < participant.size() ; i++)
    {
        if(participant[i] != completion[i]) 
        {
            answer = participant[i];
            return answer;
        }
    }
}
